src/components/Form/Checkbox.tsx

```diff
@@ -12,6 +12,17 @@ const Checkbox = ({
   onChange,
   black,
   error
+}: {
+  children: React.ReactNode | string
```

Comment:
since we use the typed children, any reason not to use `React.FC`?

Reply:
What would that do?

Reply:
basically you get the children for free. (example from the [styleguide](https://styleguide.republik.ch/dev/typescript#reactfc))
```typescript
const ArrowUp: React.FC<{
  size: number
  fill?: string 
}> = ({ size, fill, children }) => (
  <svg fill={fill} width={size} height={size} viewBox='0 0 24 24'>
    <path d='M7.41 15.41L12 10.83l4.59 4.58L18 14l-6-6-6 6z' />
    <path d='M0 0h24v24H0z' fill='none' />
    {children}
  </svg>
)
```

Reply:
I don't like it. Too much magic...
But I just found out, that "ReactNode allows multiple elements, strings, numbers, fragments, portals, …", so I can omit the `| string` part

Reply:
yes I believe you can also get rid of the array.

Reply:
We use plenty of magic in other parts of our code (graphql, etc.). But I see your concerns. What I really don't like though, is inconsistency. We can add types one way or another, but if we sometimes use generics, sometimes plain types, it kinda gets messy. Unfortunately, I can't remember reaching a consensus on that, as a team.
Can we have a quick chat on the topic tomorrow or next week?